ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminViewsRestController.java

```diff
@@ -21,48 +21,47 @@
-		this.viewValidator = viewValidator;
-		this.viewConverter = viewConverter;
-	}
+    public AdminViewsRestController(ViewService viewService,
+                                    @Qualifier("viewShaclValidator") ModelValidator viewValidator,
+                                    ViewSpecificationConverter viewConverter) {
+        this.viewService = viewService;
+        this.viewValidator = viewValidator;
+        this.viewConverter = viewConverter;
+    }
 
-	@InitBinder
-	private void initBinder(WebDataBinder binder) {
-		binder.setValidator(viewValidator);
-	}
+    @InitBinder
+    private void initBinder(WebDataBinder binder) {
+        binder.setValidator(viewValidator);
+    }
 
-	@GetMapping(value = "/eventstreams/{collectionName}/views", produces = {contentTypeTurtle, contentTypeJSONLD,
-			contentTypeNQuads})
-	public List<ViewSpecification> getViews(@PathVariable String collectionName) {
-		return viewService.getViewsByCollectionName(collectionName);
-	}
+    @GetMapping(value = "/eventstreams/{collectionName}/views", produces = {contentTypeTurtle, contentTypeJSONLD,
+            contentTypeNQuads})
+    public List<ViewSpecification> getViews(@PathVariable String collectionName) {
+        return viewService.getViewsByCollectionName(collectionName);
+    }
 
-	@ResponseStatus(value = HttpStatus.CREATED)
-	@PostMapping(value = "/eventstreams/{collectionName}/views", consumes = {contentTypeTurtle, contentTypeJSONLD,
-			contentTypeNQuads})
-	public void createView(@PathVariable String collectionName,
-						   @RequestBody @Validated Model view) {
-		viewService.addView(viewConverter.viewFromModel(view, collectionName));
-	}
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @PostMapping(value = "/eventstreams/{collectionName}/views", consumes = {contentTypeTurtle, contentTypeJSONLD,
+            contentTypeNQuads})
+    public void createView(@PathVariable String collectionName,
+                           @RequestBody @Validated Model view) {
+        viewService.addView(viewConverter.viewFromModel(view, collectionName));
+    }
 
-	@DeleteMapping("/eventstreams/{collectionName}/views/{view}")
-	public void deleteView(@PathVariable String collectionName, @PathVariable String view) {
-		ViewName viewName = new ViewName(collectionName, view);
-		viewService.deleteViewByViewName(viewName);
-	}
+    @DeleteMapping("/eventstreams/{collectionName}/views/{viewName}")
+    public void deleteView(@PathVariable String collectionName, @PathVariable String viewName) {
+        viewService.deleteViewByViewName(new ViewName(collectionName, viewName));
+    }
 
-	@GetMapping(value = "/eventstreams/{collectionName}/views/{viewName}", produces = {contentTypeTurtle,
-			contentTypeJSONLD,
-			contentTypeNQuads})
-	public ViewSpecification getViewOfCollection(@PathVariable String collectionName,
-												 @PathVariable String viewName) {
-		return viewService.getViewByViewName(new ViewName(collectionName, viewName));
-	}
```

Comment:
seems like we have a styling issue..

Reply:
Yes it does, normally this have been streamlined by now, but not all files are already stream lined